### ISSUE TITLE
Feature Suggestions

### ISSUE THREAD

**Issue by dfnsar715** [jhuckaby/Cronicle#471]
First...Cronicle is simply the best task scheduler/cron replacement - free or paid. Thanks for such an awesome program! 

I've seen the discussion about the upcoming release of Orchestra, and I just wanted to suggest two small features which I hope could be included in Orchestra (and/or a later release of Cronicle). I apologize in advance if the features are already integrated and I just missed them or they have already been requested. 

1.     I have Cronicle installed on a primary server and 5 worker servers. I have 100+ combined events scheduled to run daily on the different servers. Most of the time if an event ends with an error on any of the servers, it is clearly and easily noticeable both in the main "Completed" tab/area, as well as in the "History" section of the specific event (with a red "!Error" message in the result column). However, if a scheduled event fails to run due to  -- for instance -- one of the worker servers being offline or a drop in network connection, there is no notification that the event failed to run, at least that I have found. The only way I can tell that one or more events failed to run is because I know the specific number of daily events and can see that the normal "completed" number isn't what it should be. I then need to sort through the logs for all 100+ events to determine which event(s) on which server(s) failed to run, which sometimes takes a really long time. Is there a way to log the failure to run and classify it as an "error" or as a separate category (maybe "skipped" or something) for logging and search purposes?

2.     Unless I missed it, an event can only be assigned to one category. It would be great if there was an option to assign multiple categories to an event and then be able to sort by one or more combined categories. For instance, for me it would be great to be able to assign categories such as server name/username, work/home, backup destination, cloud service, etc. to an event and then be able to search an event by just one, some, or all assigned categories (or be able to search by different combinations of categories) .

Thanks again!

**Comment by jhuckaby** [comment]
Excellent suggestions, thank you!

An event failing to run due to server unreachable **should** show on the Activity tab (and it is logged), but I will absolutely make sure this is elevated to full error status in Orchestra, so you can find it easily.  Also, in Orchestra you will be able to show "only errors" in the history screens, to make this extra obvious.

Events assigned to multiple categories is a great idea, thank you!  Added to my TODO list.

**Comment by dfnsar715** [comment]
Awesome, thanks so much!

**Comment by mikeTWC1984** [comment]
@dfnsar715  Good catch on the "failed to start".  I'd think that should be logged in "Activity log" this could be a good place to track errors. 
Regarding the second ask - I'd rather go with adding tags/labels, if it's just about searching. Current form of category is pretty handy (kind of "directory" for jobs), plus it could be used for limiting access to users.

**Comment by jhuckaby** [comment]
Yup, Mike - I came to the same conclusion because categories have settings like default e-mail, default web hook, and default resource limits for jobs.  That would become messy if an event could have multiple categories.  Tags it is!
